src/views/edit/edit-package-view.vue

```diff
@@ -410,7 +410,7 @@
                   v-if="uuid"
                   label="UUID"
                   :value="uuid"
-                  path="/package"
+                  path="/#/package"
```

Comment:
This does not work for creating an URL which points to the REST API for example. We do not want a link to this client.